### ISSUE TITLE
Not "Connected"

### ISSUE THREAD

**Issue by noctividus** [rakibtg/docker-web-gui#52]
- The web gui is up.
- Dockers are running.
- docker ps lists a few containers
- docker image ls shows a bunch of images

The Web GUI is completely blank. Where do I start?

**Comment by rakibtg** [comment]
Can you take a look at the terminal log in the browser?

**Comment by novwhisky** [comment]
I encountered this after installing on my test server and running `node app.js` directly. The REST api [is hard-coded](https://github.com/rakibtg/docker-web-gui/blob/7a229a5adae9cb7f723b3dbdb59972d07ac2d4cb/client/src/utilities/request.js#L2) to reach out to localhost but I was requesting the client on testserver.local.

I tried using both docker-compose & docker build thinking they would prevent the host resolution issues, but the build fails on `npm install --silent` due to a non-zero exit code.
